lib/lockfile_test.go

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import (
+	"runtime"
+	"testing"
+)
+
+// Test Locking
+func TestLocking(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Log("Skipping test on Windows: cannot figure it out 01-Mar-2025")
```

Comment:
Errr!!
I'm not _as_ worried about the issues with perhaps the test coverage being missing.. does this not mean that the main locking function doesn't work on Windows?

Reply:
I guess it was my mistake in unlocking logic. Looking into it now.